### PR TITLE
chore(react-tag-picker): adds VR tests

### PR DESCRIPTION
**Pull request by bsunderhus**
## New Behavior

1. adds VR tests for TagPicker

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: apps/vr-tests-react-components/package.json [hunk 1]
```diff
@@ -66,6 +66,7 @@
     "@fluentui/react-textarea": "*",
```
Comment (fabricteam) [inline apps/vr-tests-react-components/package.json:66]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31219&commitId=b79d2458999b1ff5fc2e11de6a8900e3a470b2f3&env=prod&clientType=FLUENTUI)


  <details><summary><b>TagPicker</b> 32 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.appearance - Dark Mode.chromium.png|0| Added|
|TagPicker.appearance - High Contrast.chromium.png|0| Added|
|TagPicker.appearance - RTL.chromium.png|0| Added|
|TagPicker.appearance.chromium.png|0| Added|
|TagPicker.default - Dark Mode.chromium.png|0| Added|
|TagPicker.default - High Contrast.chromium.png|0| Added|
|TagPicker.default - RTL.chromium.png|0| Added|
|TagPicker.default open - Dark Mode.chromium.png|0| Added|
|TagPicker.default open - High Contrast.chromium.png|0| Added|
|TagPicker.default open - RTL.chromium.png|0| Added|
|TagPicker.default open.chromium.png|0| Added|
|TagPicker.default.chromium.png|0| Added|
|TagPicker.disabled - Dark Mode.chromium.png|0| Added|
|TagPicker.disabled - High Contrast.chromium.png|0| Added|
|TagPicker.disabled - RTL.chromium.png|0| Added|
|TagPicker.disabled.chromium.png|0| Added|
|TagPicker.expand icon - Dark Mode.chromium.png|0| Added|
|TagPicker.expand icon - High Contrast.chromium.png|0| Added|
|TagPicker.expand icon - RTL.chromium.png|0| Added|
|TagPicker.expand icon.chromium.png|0| Added|
|TagPicker.grouped - Dark Mode.chromium.png|0| Added|
|TagPicker.grouped - High Contrast.chromium.png|0| Added|
|TagPicker.grouped - RTL.chromium.png|0| Added|
|TagPicker.grouped.chromium.png|0| Added|
|TagPicker.secondary action - Dark Mode.chromium.png|0| Added|
|TagPicker.secondary action - High Contrast.chromium.png|0| Added|
|TagPicker.secondary action - RTL.chromium.png|0| Added|
|TagPicker.secondary action.chromium.png|0| Added|
|TagPicker.size - Dark Mode.chromium.png|0| Added|
|TagPicker.size - High Contrast.chromium.png|0| Added|
|TagPicker.size - RTL.chromium.png|0| Added|
|TagPicker.size.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: apps/vr-tests-react-components/src/stories/TagPicker.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+    ...shorthands.marginBlock('10px'),
+    ...shorthands.borderRadius(tokens.borderRadiusMedium),
+  },
+});
+
+export const Appearance = () => {
+  const styles = useStyles();
+  return (
+    <>
+      <div>
```
Comment (YuanboXue-Amber) [inline apps/vr-tests-react-components/src/stories/TagPicker.stories.tsx:156]: nit: Maybe add some paddings here too

File: apps/vr-tests-react-components/src/stories/TagPicker.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,425 @@
...
+];
+
+export const Default = (props: Pick<TagPickerProps, 'appearance' | 'size'>) => {
+  const [selectedOptions, setSelectedOptions] = React.useState<string[]>([]);
+  const onOptionSelect: TagPickerProps['onOptionSelect'] = (e, data) => {
+    setSelectedOptions(data.selectedOptions);
+  };
+
+  return (
+    <div style={{ maxWidth: 400 }}>
```
Comment (YuanboXue-Amber) [inline apps/vr-tests-react-components/src/stories/TagPicker.stories.tsx:44]: nit: let's add some padding, making it easier to see the picker's border in dark/hc background

**Review by YuanboXue-Amber — APPROVED** [review]

**Pull request merged**